src/components/ProductListPage/ProductList.css

```diff
@@ -31,7 +30,7 @@
 }
 
 .main {
-  padding: 190px 10px;
+  padding: 190px 10px 30px;
```

Comment:
Where’s the fourth value?

Reply:
don't need it if it is the same is the second